docs/types.md

```diff
@@ -84,6 +81,16 @@ The following values are allowed:
  - Negative Email - An email address preceded by a minus sign (e.g., "-alice@test.edulinq.org")
    will remove this address from the email recipients (even if they are not currently there).
    This can be useful when using course roles but you want to exclude someone.
+ - Negative Course Role - A course role preceded by a minus sign (e.g., "-student")
+   will remove all course users with that role. This can be useful when using the "\*"
+   but you want to exclude a role.
```

Comment:
The text in these bullets needs a smoothing pass.
They were fine when we were talking about emails, but now we are talking about users.
So, some of the phrasing is a little awkward.

For example, the first bullet: "Normal email addresses may be used.".
A reader could be like "wait... I thought we were talking about course users?".
Another way to phrase this bullet could be something like:
"The email of the requested user."